### ISSUE TITLE
TypeError: 'bool' object is not iterable

### ISSUE THREAD

**Issue by banbar** [ejwa/gitinspector#226]
My settings: Windows 10

Command: 
C:\OSGeo4W\apps\Python39\python.exe gitinspector.py --format=html --timeline --since=2015-09-01 -w https://github.com/banbar/GMT-456-GIS-Programming > C:\Users\banbar\Desktop/1.html

Output:
![image](https://user-images.githubusercontent.com/20798817/138090655-72d1ced0-8e3a-4715-a273-0b589f9b8217.png)

**Comment by 7henry767** [comment]
it seems only to work with python 2.x. Probably you are using 3.x

**Comment by adam-waldenberg** [comment]
Gitinspector works fine with 3.x. Though, some versions and some API's are constantly changing and breaking backward compatibility - gettext is an example of that.

**Comment by ObiWahn** [comment]
Same problem here on mac with brew.

**Comment by fourofspades** [comment]
Same issue on Windows.

**Comment by Brikwerk** [comment]
Quick note to those with this issue: I was able to get gitinspector to work on macOS with Python 3.7.13. It seems that the gettext API has changed from 3.7 to 3.9. I recommend running gitinspector again with Python 3.7 installed.

**Comment by vizcay** [comment]
The problem is that Python 3.7 or older versions is not supported for Apple M1:
```
brew install python@3.7
...
Warning: python@3.7 has been deprecated because it is deprecated upstream!
python@3.7: The x86_64 architecture is required for this software.
Error: python@3.7: An unsatisfied requirement failed this build.
```

Anybody knows how to solve this? Not a python dev unfortunately.

**Comment by adam-waldenberg** [comment]
@vizcay Yes. For now, move to Python 3.7. We need to figure out a good (not too hacky) way to support different versions of gettext - as it keeps changing :)

**Comment by vizcay** [comment]
> @vizcay Yes. For now, move to Python 3.7. We need to figure out a good (not too hacky) way to support different versions of gettext - as it keeps changing :)

Looks like it can only be done with Rosetta: https://stackoverflow.com/questions/70315418/installing-python3-7-macbook-air-m1-problem. At least with homebrew.

Thanks but I will pass.

**Comment by foxyseta** [comment]
Same here. Any updates on this?

**Comment by AugustH** [comment]
as a quick hack: in gitinspector/localization.py replace (on line 71 and line 106)
   `install(True)` 
with
   `install(None)`

tested on Python 3.10, gives some warnings (unrelated to this problem) but output seems ok.

**Comment by hlovdal** [comment]
Same on Fedora 36.

(The True->None modification worked)

**Comment by JTangming** [comment]
> 

Same on Python 3.11.3 (The True->None modification worked)

**Comment by sral97** [comment]
Same on Fedora 39 with Python 3.11.8 (The True->None modification worked)

**Comment by fuhrmanator** [comment]
Related to #213

**Comment by adam-waldenberg** [comment]
This is the same crash as #213 and #245, and it has been fixed on master with commit c3e3126, which makes the gettext install call differ per Python line.

Closing this as resolved by that fix. Thanks for the report.
